### DISCUSSION TITLE
第九站：武功山两天反穿徒步 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#38] [Announcements]
# 第九站：武功山两天反穿徒步 - forecho's Blog

引言
这次调休一个星期，最主要的是想去武功山，但是武功山最多只能玩 3 天，所以才有了前面的佛山自驾游。那个是开胃菜（为了练车），这个才是这次的主菜。

旅行主题是：徒步
时间是：3 天 2 晚
两人总共花费：2790，其中交通费花费了 2321

https://blog.forecho.com/wugong-mountain-walking-tour.html

**Comment by holmesian** [comment]
考个古。
